modules/search/class.jetpack-search-widget-filters.php

```diff
@@ -478,27 +448,31 @@ class="widefat"
 	function render_widget_search_form( $instance ) {
```

Comment:
My [sorting UI PR](https://github.com/Automattic/jetpack/pull/8542/) also does form field injection, but does so using Javascript. After merging this we should probably choose which technique we prefer.

An advantage of my other technique (which you could also implement in PHP) is that if another plugin has injected a field with the same name, it reuses it. It is conceivable that other plugins have already added post type filtering to the search form HTML.